src/Features/Core/Portable/UseNullPropagation/AbstractUseNullPropagationCodeFixProvider.cs

```diff
@@ -34,7 +38,11 @@ internal abstract class AbstractUseNullPropagationCodeFixProvider<
         where TMemberAccessExpression : TExpressionSyntax
         where TConditionalAccessExpression : TExpressionSyntax
         where TElementAccessExpression : TExpressionSyntax
+        where TElementBindingExpression : TExpressionSyntax
+        where TElementBindingArgumentList : SyntaxNode
```

Comment:
I'm just going to say this list of where clauses is terrifyingly long.

Reply:
:)